luigi/contrib/external_program.py

```diff
@@ -60,6 +64,17 @@ class ExternalProgramTask(luigi.Task):
     """
 
     capture_output = luigi.BoolParameter(default=True, significant=False, positional=False)
+    tracking_url_pattern = luigi.Parameter(
+        default="", significant=False, positional=False,
+        description="Regex pattern used for searching URL in the logs of the external program")
+    """
+    Regex pattern used for searching URL in the logs of the external program.
+
+    If a log line matches the regex, the first group in the matching is set as the tracking URL
+    for the job in the web UI. Example: 'Job UI is here: (https?://.*)'.
+
+    Default value is an empty string, so URL tracking is not performed.
+    """
```

Comment:
Does this docs render nice?

Reply:
Hmm, how can I check it?

Reply:
I recently wrote about it here: https://github.com/spotify/luigi/pull/2548#issuecomment-465763391